package.json

```diff
@@ -157,7 +157,7 @@
     "it-pushable": "^3.0.0",
     "it-stream-types": "^1.0.4",
     "uint8arraylist": "^2.1.1",
-    "uint8arrays": "^3.0.0",
+    "uint8arrays": "^3.1.0",
```

Comment:
This is an in-range semver update.  No need to specify the minor, just remove your lockfile and you will get the latest version.